README.md

```diff
@@ -30,25 +30,65 @@ await compiler.save('./features/dist/login.feature', ast, {
 ```typescript
 'use strict';
 import {load, process, save} from "gherking";
-import Template = require("gpc-template");
+import Filter = require("gpc-filter");
 
 let ast = await load("./features/src/login.feature");
 ast = process(
     ast,
-    new Template({
+    new Filter({
         // config
     })
 );
 await save('./features/dist/login.feature', ast, {
     lineBreak: '\r\n'
 });
 ```
+
+## Configuration
+
+The precompiler accepts the following configuration:
+
+| Option | Type | Description | Necessity |
+|:------:|:----:|:------------|:----------|
+| `includeScenarioTags` | `Array` | Scenarios with the given tag will be included in the result | Optional |
+| `excludeScenarioTags` | `Array` | Scenarios with the given tag will be excluded from the result | Optional |
+| `includeRuleTags` | `Array` | Rules with the given tag will be included in the result | Optional |
+| `excludeRuleTags` | `Array` | Rules with the given tag and all their elements will be excluded from the result | Optional |
+
+## Example
+Excluding scenarios with @wip or @exclude tags
```

Comment:
I'd either rephrase it: "Keeping scenarios with `@current` tag", or set a filter what means this condition.